### ISSUE TITLE
Error while generating signed APK: "Error:(5) Error: "forecast_api_key" is not translated in "nl" (Dutch) [MissingTranslation]"

### ISSUE THREAD

**Issue by ordago** [NielsMasdorp/Speculum-Android#38]
```
Error:(5) Error: "forecast_api_key" is not translated in "nl" (Dutch) [MissingTranslation]
Error:(7) Error: "static_maps_api_key" is not translated in "nl" (Dutch) [MissingTranslation]
```

The error doesn't happen when creating a non signed APK.

I did this:

```
<string name="forecast_api_key" translatable="false">myKey</string>
<string name="static_maps_api_key" translatable="false"></string>
```
This way it doesn't complain and the app works.

**Comment by NielsMasdorp** [comment]
Ah, thanks for the headsup! To be honest I never built a signed apk for
speculum 😁

**Comment by NielsMasdorp** [comment]
Fixed in PR #40
